bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/ImageList.java

```diff
@@ -445,4 +457,22 @@ public int size () {
 	return result;
 }
 
+public static long win32_getHandle(ImageList imageList, int zoom) {
```

Comment:
Shouldn't the `getHandle()` method be removed in favor of the `win32_getHandle()` method? It should even be possible to replace the existing `getHandle()` method with one accepting a zoom parameter, as this class is internal and its interface may be changed.

Reply:
Yes, definitely. I moved the new logic into getHandle and adapted all calls accordingly